examples/automm/object_detection/coco_eval.py

```diff
@@ -0,0 +1,21 @@
+from autogluon.multimodal import MultiModalPredictor
+
+def test_coco_evaluation(checkpoint_name = "faster_rcnn_r50_fpn_2x_coco"):
+    predictor = MultiModalPredictor(
+        label="rois",
+        hyperparameters={
+            "model.mmdet_image.checkpoint_name": checkpoint_name,
+            "env.num_gpus": 1,
+        },
+        pipeline="object_detection",
+    )
+
+    import time
+    start = time.time()
+    predictor.evaluate('coco17/annotations/instances_val2017.json')
+    print("time usage: %.2f" % (time.time() - start))
+
+if __name__ == "__main__":
```

Comment:
This code block is not necessary. If you'd like to execute a certain test-case, you may directly use pytest.

Reply:
Okay, I thought it's in test-case but it's actually in example.